src/Microsoft.PowerShell.ConsoleGuiTools/OutConsoleGridviewCmdletCommand.cs

```diff
@@ -46,21 +46,9 @@ public class OutConsoleGridViewCmdletCommand : PSCmdlet, IDisposable
         /// Get or sets a value indicating whether the selected items should be written to the pipeline
         /// and if it should be possible to select multiple or single list items.
         /// </summary>
-        [Parameter(ParameterSetName = "OutputMode")]
+        [Parameter()]
         public OutputModeOption OutputMode { set; get; }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether the selected items should be written to the pipeline.
-        /// Setting this to true is the same as setting the OutputMode to Multiple.
-        /// </summary>
-        [Parameter(ParameterSetName = "PassThru")]
-        public SwitchParameter PassThru
```

Comment:
This is a breaking change. It'd be nice to slowly phase it out by writing a warning that it is no longer needed.

Or some indication in the help content that it's no longer needed and is the default behavior.

Reply:
But I guess it was always a preview so we can break